### DISCUSSION TITLE
Can I add title in Header of Items List?

### DISCUSSION THREAD

**Discussion by Saleh-Salem** [rollbar/rollbar-gem#1097] [Q&A]
Can I edit title header or customize the Header on the Items List Page?
<img width="1066" alt="Screen Shot 2022-06-30 at 2 35 49 PM" src="https://user-images.githubusercontent.com/22606707/176669010-e7729175-95f1-4301-a3c3-17f1fe193b47.png">

**Comment by brianr** [comment]
Yes, a little - you can show/hide from those columns:

1. Click the icon next to Items 
![image](https://user-images.githubusercontent.com/12275/176740213-20c13522-481c-4575-ae5e-b2df4ef948bf.png)

2. Toggle the columns you don't want
![image](https://user-images.githubusercontent.com/12275/176740333-4b293006-2896-4630-95b1-b98de2b03d65.png)

Is that what you're looking for?

**Reply by Saleh-Salem** [reply]
Thanks, but I mean programmatically send with data from my app.
